bin/setup

```diff
@@ -15,6 +15,7 @@ chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
+  system! 'gem install mailcatcher --conservative'
```

Comment:
@Kolisnyk it's better to just put instructions for developers into README.md file.
We don't need `mailcatcher` in production.